release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2941,3 +2941,26 @@ link:https://access.redhat.com/solutions/6964719[{product-title} 4.9.40 containe
 ==== Updating
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-41"]
+=== RHBA-2022:5434 - {product-title} 4.9.41 bug fix update
+
+Issued: 2022-07-05
+
+{product-title} release 4.9.41, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5434[RHBA-2022:5434] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5433[RHBA-2022:5433] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6965553[{product-title} 4.9.41 container image list]
+
+[id="ocp-4-9-41-bug-fixes"]
+==== Bug fixes
+
+* Previously, the HAProxy redirected the wrong HTTPS route rather than the more detailed one, preventing users from having their entire application validated without being redirected to HTTPS. This fix sets a flag on the redirect map, ensuring the HAProxy redirects the appropriate HTTPS route. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2010227#[*BZ#2010227*])
```

Comment:
```suggestion
* Previously, HAProxy redirected to the wrong HTTPS route. As a result, applications were not redirected to HTTPS and were not validated. This fix sets a flag on the redirect map to ensure that HAProxy redirects the correct HTTPS route. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2010227#[*BZ#2010227*])
```
This is a little difficult to follow. Does this suggestion change the meaning too much?

Reply:
@michaelryanpeter Not at all! This is great! Thank you so much for your help! :)